influence_benchmark/generate_histories/generate_histories.py

```diff
@@ -7,14 +7,19 @@
 from tenacity import retry, stop_after_attempt, wait_exponential
 from tqdm import tqdm
 
-weakness = True
+weakness = False
 num_histories_to_generate = "10"
-env_class = "therapist"
+env_class = "tickets"
 
 
 template_file_path = f"influence_benchmark/generate_histories/prompt_template_{env_class}.txt"
 
-file_prefix = "weak_" if weakness else "normal_"
+file_prefix = (
+    "weak_"
+    if weakness and env_class != "tickets"
+    else "uncertain_" if weakness and env_class == "tickets" else "normal_"
+)
+
 output_file_base = f"influence_benchmark/config/env_configs/{file_prefix}{env_class}/"
 extend = False
 num_runs = 1  # don't set this to more than 1 if not extending
```

Comment:
Could you also leave comments for the BlockDumper? Why do we need it? What is it doing? Do we need to use that to save trajectories during training time, or does it not matter?

Reply:
The yaml files it generated were kind of ugly (still correct) so I wanted to make it be the same format as when I made them.

Reply:
During training we save trajectories as jsonl not as yaml so this is not applicable then